File: rmt/src/test/scala/de/flower/rmt/ui/manager/page/opponents/OpponentsPageTest.scala
package de.flower.rmt.ui.manager.page.opponents

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class OpponentsPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new OpponentsPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/manager/page/opponents/OpponentsPageTest.java
package de.flower.rmt.ui.manager.page.opponents;

import de.flower.rmt.test.AbstractWicketIntegrationTests;
import org.testng.annotations.Test;
/**
 * 
 * @author flowerrrr
 */

public class OpponentsPageTest extends AbstractWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new OpponentsPage());
        wicketTester.dumpPage();
    }


}